devtools/maven/src/main/java/io/quarkus/maven/CreateProjectMojo.java

```diff
@@ -144,6 +160,26 @@ public void execute() throws MojoExecutionException {
         }
     }
 
+    private void createMavenWrapper() {
+        try {
+            executeMojo(
+                    plugin(
+                            groupId("io.takari"),
+                            artifactId("maven"),
+                            version("0.7.4")),
+                    goal("wrapper"),
+                    configuration(
+                            element(name("maven"), "3.5.4")),
```

Comment:
This shouldn't be hardcoded. I know @cescoffier already passes a few properties to the mojo using a properties file, we should add the 2 versions there.

Reply:
Makes sense of course. I think I see where those properties are, so I will make sure to use them